rosbag2_cpp/include/rosbag2_cpp/clocks/time_controller_clock.hpp

```diff
@@ -102,6 +102,16 @@ class TimeControllerClock : public PlayerClock
   ROSBAG2_CPP_PUBLIC
   bool is_paused() const override;
 
+  /**
+   * Change the current ROS time to an arbitrary time.
+   * This will wake any waiting `sleep_until`.
+   * Note: this initial implementation does not have any jump-callback handling,
+   * so it is not safe to use during playback, only for reconfiguration for clock reuse.
```

Comment:
> what does "reconfiguration for clock reuse" mean?

I mean - "to reconfigure the clock so that you can use the same clock instance again, with all current settings, at a different starting time". Is that wording any more clear?

>  should we then actively prevent this from happening?

No - because the clock has no concept of "playback". This is a "usage hint" to the `Player`. We could restrict its usage to "only while paused" - but that doesn't really help anything, the `Player`'s current queueing mechanism will still be completely broken if `jump` is used while the queue is active.